vector/src/main/res/layout/fragment_ftue_auth_use_case.xml

```diff
@@ -0,0 +1,167 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="?android:colorBackground"
+    android:paddingTop="@dimen/layout_vertical_margin"
+    android:paddingBottom="@dimen/layout_vertical_margin">
+
+    <androidx.constraintlayout.widget.Guideline
+        android:id="@+id/useCaseGutterStart"
+        android:layout_width="wrap_content"
+        android:layout_height="match_parent"
+        android:orientation="vertical"
+        app:layout_constraintGuide_percent="@dimen/ftue_auth_gutter_start_percent" />
+
+    <androidx.constraintlayout.widget.Guideline
+        android:id="@+id/useCaseGutterEnd"
+        android:layout_width="wrap_content"
+        android:layout_height="match_parent"
+        android:orientation="vertical"
+        app:layout_constraintGuide_percent="@dimen/ftue_auth_gutter_end_percent" />
+
+    <ImageView
+        android:id="@+id/useCaseHeaderIcon"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="52dp"
```

Comment:
We do not have a scrollview here, I am afraid the whole screen will render badly on small screens (did not check)

Reply:
yep it's broken...

| 240 DPI - nexus one | 
| --- | 
![Screenshot_20220113_110500](https://user-images.githubusercontent.com/1848238/149318841-6107e4b8-c32c-4aed-8835-c68f85a80120.png)

Reply:
b371e24d9fd5ef88c976191134db7e0ddde4d820

PR description screenshots updated